src/components/ui-shell/_header.scss

```diff
@@ -29,6 +29,15 @@
     border: mini-units(0.5) solid transparent;
   }
 
+  .#{$prefix}--header__action > svg[data-navigation-menu-panel-collapse-icon],
```

Comment:
Same note as other PRs, is there any way to do this without tying the selector to the data attribute?

Reply:
got it, should be resolved now!